php/main.php

```diff
@@ -9,76 +8,79 @@
 $password = "root";
 $database = "homeless";
 
+$conn = mysqli_connect($servername, $username, $password, $database);
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST'){
+
+    if (isset($_POST['finger_status'])){
 
-if ($_SERVER['REQUEST_METHOD'] === 'POST') {
-  if (isset($_POST['finger_status'])){
-    $status = $_POST['finger_status'];
-    if ($status === "success"){
-      $conn = mysqli_connect($servername, $username, $password, $database);
-      $sql = "UPDATE coreData SET state='3' WHERE State='2'";
-      $conn->query($sql);
+        $status = $_POST['finger_status'];
```

Comment:
We should lowercase, especially if we are comparing it

Reply:
Hmmm, shouldn't matter because we send those status, but will do that just in case we change it in the future.